### PR TITLE
RHDEVDOCS-3813 - added steps to install the operator using CLI

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.9, 4.10
JIRA issue: https://issues.redhat.com/browse/RHDEVDOCS-3813
Preview pages: https://deploy-preview-42879--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/installing-openshift-gitops

SME+QE review: @jannfis
Peer-review: @Srivaralakshmi

### CODE REVIEW

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc [hunk 1]
```diff
@@ -32,5 +32,5 @@ To login with your {product-title} credentials, select the `LOG IN VIA OPENSHIFT
 
 [NOTE]
 ====
-You cannot create two Argo CD CRs in the same namespace.
+You cannot create two Argo CD CRs in the same namespace.logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
```
Comment (jannfis) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:35]: That looks weird to me.

**Review by jannfis — COMMENTED** [review]
LGTM with a minor concern

File: modules/installing-gitops-operator-using-cli.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace 
+----
+. Apply the `Subscription` to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f openshift-gitops-sub.yaml
+----
+. After the installation is complete, the following pods will appear:
```
Comment (Srivaralakshmi) [inline modules/installing-gitops-operator-using-cli.adoc:33]: @DebarghoGhosh I like the verification step :)

```suggestion
. After the installation is complete, ensure that all the pods in the `openshift-gitops` namespace are running:
```

File: modules/installing-gitops-operator-using-cli.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
++
```
Comment (Srivaralakshmi) [inline modules/installing-gitops-operator-using-cli.adoc:34]:
```suggestion
+
.Example output
```

File: modules/installing-gitops-operator-using-cli.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+[source,terminal]
+----
+$ oc get pods -n openshift-gitops
```
Comment (Srivaralakshmi) [inline modules/installing-gitops-operator-using-cli.adoc:37]:
```suggestion
+
[source,terminal]
----
$ oc get pods -n openshift-gitops
----
```

File: modules/installing-gitops-operator-using-cli.adoc [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+NAME                                                      	READY   STATUS	RESTARTS   AGE
```
Comment (Srivaralakshmi) [inline modules/installing-gitops-operator-using-cli.adoc:38]:
```suggestion
+
The output, which takes a few minutes to display, verifies that the {gitops-title} Operator is now installed in the `openshift-operators` default target namespace:
+
.Example output
[source,terminal]
----
NAME                                                      	READY   STATUS	RESTARTS   AGE
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@DebarghoGhosh Good work, left a few suggestions, PTAL. Thanks!

File: modules/installing-gitops-operator-using-cli.adoc [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+[role="_abstract"]
+You can install {gitops-title} Operator from the OperatorHub using the CLI.
+
+.Procedure
+
+. Create a Subscription object YAML file to subscribe a namespace to the {gitops-title}, for example, `sub.yaml`:
+
+.Example Subscription
++
```
Comment (Srivaralakshmi) [inline modules/installing-gitops-operator-using-cli.adoc:17]:
```suggestion

```

File: modules/installing-gitops-operator-using-cli.adoc [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  name: openshift-gitops-operator <2>
+  source: redhat-operators <3>
+  sourceNamespace: openshift-marketplace <4> 
+----
+<1> Specify the channel name from where you want to subscribe the Operator.
+<2> Specify the name of the Operator to subscribe to.
+<3> Specify the name of the CatalogSource that provides the Operator.
+<4> The namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.
+
+. Apply the `Subscription` to the cluster:
```
Comment (Srivaralakshmi) [inline modules/installing-gitops-operator-using-cli.adoc:37]: Suggestion: The output is not coming good. Please check and rectify. This is coming as step 1 and all the steps are spilling over to the right pane.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@DebarghoGhosh PTAl and let me know once you address the comments. Will take a final look.

**Review by Srivaralakshmi — APPROVED** [review]
@DebarghoGhosh Good work! Thanks for the changes. LGTM.

**Pull request merged**